### DISCUSSION TITLE
Calculate math with addition, submission or calculated field

### DISCUSSION THREAD

**Discussion by maurerle** [PostgREST/postgrest#3520] [Q&A] [answered]
### Description of issue

I was wondering what the best way of creating a query like:

`select temp_air-273.15 as temp_celsius from weather` would be.

`select wind_meridional+wind_zonal as temp_celsius from weather`

If I understood correctly, there is currently no way to execute the above query using postgREST?
In this case I would like to vote to add a way to apply mathematical operations in the select column :)

I did not find a similar issue to this yet :)

**Comment by wolfgangwalther** [comment, marked as answer]
The calculation you are doing looks like you don't need dynamic calculations. I fact you are trying to represent a value stored in the database as Kelvin as a value in Celsius on the api level, right? (Or maybe the other way around, but the idea is the same)

This looks like a clear-cut case of [domain representations](https://postgrest.org/en/v12/references/api/domain_representations.html). The big advantage: You can also hardcode the other way back, when updating the column.

**Reply by maurerle** [reply]
Hm yes, so the application developer also needs to have access to create new store procedures/domains/views.

That is fine - I was just wondering if this is possible without telling the database more about my applications requirements :)
Thank you very much :)
